jobs/aws/eks-distro-build-tooling/golang-1.16-presubmits.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+# IMPORTANT: This Prowjob was generated by running `make prowjobs -C templater`.
+# DO NOT EDIT THIS FILE BY HAND! Refer to docs/prowjobs.md for instructions on
+# how to add a new Prowjob or update an existing Prowjob.
+################################################################################
+
+presubmits:
+  aws/eks-distro-build-tooling:
+  - name: golang-1.16-tooling-presubmit
+    always_run: false
+    run_if_changed: "projects/golang/go/1.16/.*"
+    max_concurrency: 10
+    cluster: "prow-presubmits-cluster"
+    skip_report: false
+    decoration_config:
+      gcs_configuration:
+        bucket: s3://prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp
+        path_strategy: explicit
+      s3_credentials_secret: s3-credentials
+    spec:
+      serviceaccountName: presubmits-build-account
+      automountServiceAccountToken: false
+      containers:
+      - name: build-container
+        image: public.ecr.aws/amazonlinux/amazonlinux:2
+        command:
+        - bash
+        - -c
+        - >
```

Comment:
Do we not need to in yum install make?

Reply:
good catch, I had forgotten to rebase prior to copying over the 1.15 job. Let me fix this.

Reply:
alright updated